external-docs/docs/guides/command-line/inspect-command.md

```diff
@@ -1,3 +1,939 @@
+        - Number of Concepts:
+        - Number of Duplicates:
+        - First 10 Concepts:
+        - Last 10 Concepts:
+
+- The code list has the following concepts information:
+        - Concepts hierarchy depth:
+        - Concepts hierarchy:
+```
+
+* `This file is a code list` - whether the input CSV-W is of type data cube or code list.
+  
+* `The code list has the following catalog metadata`:
+  
+  | Field         | Description                                                         |
+  | ------------- | ------------------------------------------------------------------- |
+  | Title         | Title of the data set.                                              |
+  | Label         | Label of the data set.                                              |
+  | Issued        | Data set issued timestamp in the `YYYY-DD-MM'T'HH:mm:ssZ` format.   |
+  | Modified      | Data set modified timestamp in the `YYYY-DD-MM'T'HH:mm:ssZ` format. |
+  | License       | License information.                                                |
+  | Creator       | Creator information web page url.                                   |
+  | Publisher     | Publisher information web page url.                                 |
+  | Landing Pages | List of web urls of landing pages.                                  |
+  | Themes        | List of web urls of theme information.                              |
+  | Keywords      | List of keywords.                                                   |
+  | Contact Point | Contact information.                                                |
+  | Identifier    | Data set identifier.                                                |
+  | Comment       | Any comments associated with the data set.                          |
+  | Description   | Formatted description of the data set.                              |
+
+* `The code list has the following dataset information`:
+
+  | Field                 | Description                                     |
+  | --------------------- | ----------------------------------------------- |
+  | Number of Concepts    | Number of concepts (i.e. rows) in the data set. |
+  | Number of Duplicates  | Number of duplicated concepts in the data set.  |
+  | First 10 Observations | Sample consists of first 10 concepts.           |
+  | Last 10 Observations  | Sample consists of last 10 concepts.            |
+
+* `The code list has the following concepts information`:
+
+  | Field                    | Description                                              |
+  | ------------------------ | -------------------------------------------------------- |
+  | Concepts hierarchy depth | The hierachy depth of the code list concepts.            |
+  | Concepts hierarchy       | Tree representation of the code list concepts hierarchy. |
+
+## Inspect command errors
+
+When errors are encountered, please refer to the [errors guide](../guides/errors/index.md) for help on understanding and resolving them.
```

Comment:
Weren't you going to create an inspect command sub-setion in errors? I guess that's what you're still working on?